main.py

```diff
@@ -0,0 +1,19 @@
+from managers.PrinterManager import PrinterManager
```

Comment:
According to [pep8](https://peps.python.org/pep-0008/#package-and-module-names) module naming convention, you'd better name modules like_this and not LikeThis. So a pythonic way to do it would be:

```python
from models.inkjet_printer import InkjetPrinter
```